addons/website_blog/views/website_blog_components.xml

```diff
@@ -5,9 +5,10 @@
 <!-- ======   Template: Blogs Navbar   =========================================
 ============================================================================ -->
 <template id="blogs_nav" name="Blogs Bar Template">
-    <nav t-attf-class="navbar navbar-expand-sm navbar-light #{additionnal_classes}">
-        <div class="container">
-            <ul class="navbar-nav" t-if="len(blogs) > 1">
+    <nav t-attf-class="navbar navbar-expand-sm navbar-light pt-4 pb-0 px-0 #{additionnal_classes}">
+        <div class="o_website_blog_nav_container container flex-nowrap w-100">
```

Comment:
Thanks it's better! You can find the [diff here.](https://github.com/odoo/odoo/compare/7b8f3010af608166460cea43c3b49cb4ce70666c..e7bbc94197594a7c1f820f1c659eb98317cfbe26)

The classes in the `<t t-set="_classes" t-valuef=""/>` are not useful anymore but I think it's safer to keep the t-set here in case of xpath. 

As RDE pointed out it might be worth keeping the classes here to take 0 risks but since it's a t-valuef with styling classes it seems unlikely to break unless someone created an xpath removing some of these classes in this specific attribute. 🤔